partner_internal_code/models/res_partner.py

```diff
@@ -5,9 +5,8 @@
 from odoo import fields, models, api
 
 
-class partner(models.Model):
-
-    """"""
+class Partner(models.Model):
+    
```

Comment:
a pylint error appears for to many empty lines. 2 lines after class Partner line and it should be only one in this case.